tune/tune.c

```diff
@@ -97,18 +97,18 @@ static int exfat_set_volume_label(struct exfat_blk_dev *bd,
 {
 	struct exfat_dentry vol;
 	int nbytes;
-	__u16 volume_label[11];
+	__u16 volume_label[VOLUME_LABEL_MAX_LEN];
 	int volume_label_len;
 
 	volume_label_len = exfat_utf16_enc(label_input,
-			volume_label, sizeof(volume_label));
+			volume_label, VOLUME_LABEL_MAX_LEN);
```

Comment:
the 3rd argument should be the size of volume_label, instead of the length.

Reply:
They're the same value
sizeof will report the length of the array ( hard coded as 11 )